card/mod/standard/set/all/rich_html/editing.rb

```diff
@@ -187,7 +187,7 @@ def rename_referer_effect_and_option
     [effect, option]
   end
 
-  view :edit_type, perms: :update do
+  view :edit_type, cache: :never, perms: :update do
```

Comment:
as I mentioned in slack editing the structure doesn't work anymore because of caching issues.
The spirit of these changes was: we don't win much by caching these views but loose a lot by caching them wrong